app/config/Constants.scala

```diff
@@ -26,4 +26,6 @@ object Constants {
 
   val delimitedPSA = "DELIMITED_PSAID"
   val detailsNotFound = "no match found"
+
+  val MaxSchemeBankAccounts = 10
```

Comment:
Values start with lowercase letters

Reply:
It's uppercase so I can use it in a pattern match statement without prefixing with `Constants.` but can change it to lowercase if we don't like doing that

Reply:
If you do the refactor to the navigator you won't need this anymore